src/lib/components/badge.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  export let type: BadgeTypes;
+  type BadgeTypes = keyof typeof Badges;
+
+  // We need to keep the entire classname intact so tailiwnd
```

Comment:
I might just use CSS classes here like we do in the rest of the application.

Also, there is a small typo.